mobile/lib/shared/models/immich_logger_message.model.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:hive/hive.dart';
+
+part 'immich_logger_message.model.g.dart';
+
+@HiveType(typeId: 3)
+class ImmichLoggerMessage {
+  @HiveField(0)
+  String message;
+
+  @HiveField(1, defaultValue: "INFO")
+  String level;
```

Comment:
Can't we use the logging library's `Level` enum here?

Reply:
that would be perfect! otherwise an int? (enums should map to an int in Dart?)

Reply:
Unfortunately, Hive doesn't make that easy for you, since the type is from the external library.

https://docs.hivedb.dev/#/custom-objects/type_adapters

Reply:
Ah, that's annoying, but OK 😛